product/host.go

```diff
@@ -72,7 +73,7 @@ func hostRunners(ctx context.Context, os string, redactions []*redact.Redact, l
 		host.NewOS(os, redactions),
 		host.NewDisk(redactions),
 		host.NewInfo(redactions),
-		host.Memory{},
+		host.NewMemoryWithContext(ctx, runner.Timeout(30*time.Second)),
```

Comment:
Not a big deal here, but if we end up making 30 seconds a common default across host runners, I think it may be worth making it a constant so that it's easier to update it.

Reply:
I'll make a todo note as we keep implementing timeouts in the runners. Ideally the defaults would be sourced from the product via the agent, but for any one-offs then a const is for sure the way to go.